refl1d/reflectivity.py

```diff
@@ -213,20 +217,26 @@ def magnetic_amplitude(kz,
     if np.isscalar(sigma):
         sigma = sigma*np.ones(n-1, 'd')
 
-    #kz = -kz
-    #depth, rho, irho, sigma, rhoM, thetaM = [v[::-1] for v in (depth, rho, irho, sigma, rhoM, thetaM)]
-    depth, rho, irho, sigma = [_dense(a, 'd') for a in (depth, rho, irho, sigma)]
-    #np.set_printoptions(linewidth=1000)
-    #print(np.vstack((depth, np.hstack((sigma, np.nan)), rho, irho, rhoM, thetaM)).T)
+    # kz = -kz
+    # depth, rho, irho, sigma, rhoM, thetaM = [v[::-1] for v in (depth, rho, irho, sigma, rhoM, thetaM)]
+    depth, rho, irho, sigma = [_dense(a, 'd')
+                                      for a in (depth, rho, irho, sigma)]
```

Comment:
Maybe indent as:
```
LHS = [
    list contents]
```
or
```
LHS = [
    list contents,
]
```
I find this makes for less work when changing LHS. You don't need to reindent the continuation lines.